ccxt.d.ts

```diff
@@ -84,7 +84,9 @@ declare module 'ccxt' {
         feeSide?: string | undefined;
         precision: {
             amount: number | undefined,
-            price: number | undefined
+            price: number | undefined,
+            base: number | undefined,
```

Comment:
actually, I meant optional type like this

```Javscript
number?: number | string
```
since most exchanges rn don't have these (not even with an undefined value)

Reply:
ah, your' right.